### DISCUSSION TITLE
How to change language interface for Apache NetBeans IDE 19

### DISCUSSION THREAD

**Discussion by Empers100** [apache/netbeans#6769] [General]
In NetBeans IDE 8.2 i changed the netbeans.conf and the netbeans_default_options parametr and added the line: -J-Duser.language=en -J-Duser.region=US. In version 19 this option does not work. How to change the language interface of the program?

**Comment by JoachimRohde** [comment]
That's currently not possible. See also [this](https://github.com/apache/netbeans/issues/6402) ticket.
